cmake/RPackageInstall.cmake.in

```diff
@@ -0,0 +1,34 @@
+# Commands to install the R package as a CMake install target
+
+function(check_call)
+  set(cmd COMMAND)
+  cmake_parse_arguments(
+    PARSE_ARGV 0
+    CALL_ARG "" "" "${cmd}"
+  )
+  string(REPLACE ";" " " commands "${CALL_ARG_COMMAND}")
+  message("Command: ${commands}")
+  execute_process(COMMAND ${CALL_ARG_COMMAND}
+                  OUTPUT_VARIABLE _out
+				  ERROR_VARIABLE _err
+				  RESULT_VARIABLE _res)
+  if(NOT "${_res}" EQUAL "0")
+    message(FATAL_ERROR "out: ${_out}, err: ${_err}, res: ${_res}")
```

Comment:
Previously, CMake would run commands like `R CMD INSTALL` but would not check if the command actually succeeded or not. Now CMake will check the result of the command. This change will help us debug install issues in the coming future.

Reply:
oh great!